src/views/Farms/components/Table/Table.tsx

```diff
@@ -0,0 +1,291 @@
+import React, { useRef, useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { RowType, useTable, ChevronDownIcon } from '@pancakeswap-libs/uikit'
+import BigNumber from 'bignumber.js'
+import useI18n from 'hooks/useI18n'
+
+import Row, { RowData } from './Row'
+import ScrollBar from '../ScrollBar'
+import { ColumnsDef } from '../types'
+import Cell from '../Cell'
+
+export interface ITableProps {
+  data: RowData[]
+}
+
+const Container = styled.div`
+  padding: 1rem;
+  box-shadow: 0px 2px 12px -8px rgba(25, 19, 38, 0.1), 0px 0px 1px rgba(25, 19, 38, 0.15);
+  border-radius: 32px;
+  margin: 1rem 0rem;
+  width: 100%;
+  background: ${(props) => props.theme.card.background};
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    padding: 1.5rem;
+  }
+`
+
+const TableWrapper = styled.div`
+  overflow: auto;
+  max-height: 600px;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+`
+
+const StyledTable = styled.table`
+  border-collapse: collapse;
+  font-size: 0.9rem;
+  border-radius: 4px;
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+`
+
+const TableHead = styled.thead`
+  & tr {
+    background-color: ${(props) => props.theme.colors.tertiary};
+    border-radius: 4px;
+    color: #8f80ba;
+    font-weight: 700;
+    font-size: 0.75rem;
+    text-transform: capitalize;
+    cursor: pointer;
+
+    & th {
+      background-color: ${(props) => props.theme.colors.tertiary};
+      position: sticky;
+      top: 0;
+      padding: 1px 0px;
+      z-index: 100;
+
+      &:first-child {
+        padding-right: 0.8rem;
+      }
+    }
+  }
+
+  & input {
+    font-weight: 700;
+    border: 1px solid #cccccc;
+    border-radius: 10px;
+    padding: 4px;
+    width: 80%;
+    outline-width: 0;
+  }
+
+  & .bold {
+    color: #7645d9;
+  }
+`
+
+const TableBody = styled.tbody`
+  & tr {
+    td {
+      font-size: 0.875rem;
+      vertical-align: middle;
+      padding-right: 1rem;
+
+      &:first-child {
+        position: sticky;
+        left: 0;
+        padding-right: 0;
+        background: ${(props) => props.theme.card.background};
+
+        > div {
+          padding-right: 0;
+        }
+      }
+    }
+  }
+`
+
+const ArrowIcon = styled(ChevronDownIcon)`
+  margin-left: 0.375rem;
+  transition: 0.5s;
+
+  &.toggle {
+    transform: rotate(180deg);
+  }
+`
+const CellInner = styled.div`
+  padding: 0.3125rem 0rem;
+  display: flex;
+  width: 100%;
+  align-items: center;
+  padding-right: 1rem;
+`
+
+const TableContainer = styled.div<{ showGradient: boolean }>`
+  position: relative;
+  &:after {
+    transition: 0.3s;
+    opacity: ${(props) => (props.showGradient ? '1' : '0')};
+    display: block;
+    content: "";
+    width: 0.875rem;
+    height: 100%;
+    position: absolute;
+    top: 0;
+    right: 0;
+    background: linear-gradient(270deg, #E9EAEB 0%, rgba(233, 234, 235, 0) 100%);
```

Comment:
Is this gradient new? Is it in the theme? How about dark mode?

Reply:
This is not in the theme and it's new gradient.
It was same in the dark mode.